Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-count.txt

MongoDB\\Collection::count()
============================

.. meta::

>    :description: Count documents matching filter criteria using `MongoDB\Collection::count()`, with options for collation, hint, limit, and more.

Reviewer: ```suggestion
   :description: Count documents matching a filter criteria with the MongoDB PHP Library, with options for collation, hint, limit, and more.
```
